src/test/resources/com/puppycrawl/tools/checkstyle/checks/metrics/classfanoutcomplexity/InputClassFanOutComplexityExtendsImplementsAnnotations.java

```diff
@@ -0,0 +1,32 @@
+package com.puppycrawl.tools.checkstyle.checks.metrics.classfanoutcomplexity;
```

Comment:
Please do not mix inputs, there should be one input per new token to make test cases clear.

Reply:
fixed